portfolio/src/main/webapp/script.js

```diff
@@ -31,12 +31,21 @@ function getMessage() {
 }
 
 
-function getMessageJSON() {
+function getCommentJSON() {
   fetch('/data')  // sends a request to /data
   .then(response => response.json()) // parses the response as JSON
-  .then((message) => { // now we can reference the fields in myObject!
-    console.log(message[0]);
-    console.log(message[1]);
-    console.log(message[2]);
+  .then((comments) => { // now we can reference the fields in myObject!
+
+    // Build list of comments
+    const commentsEl = document.getElementById('comment-container');
+    comments.Comments.forEach((line) => {
+      historyEl.appendChild(createListElement(line));
```

Comment:
Did you mean `commentsEl`? Doesn't look like this will work.

Reply:
Oh right thank you!